unit-tests/live/frames/test-fps.py

```diff
@@ -126,7 +126,7 @@ def frame_cb(frame):
         else:
             fps = measure_fps(cs, cp, time_to_test_fps[i])
             log.i("Requested fps: {:.1f} [Hz], actual fps: {:.1f} [Hz]. Time to first frame {:.6f}".format(requested_fps, fps, first_frame_seconds))
-            delta_Hz = requested_fps * 0.05 # Validation KPI is 5%
+            delta_Hz = requested_fps * (0.10 if requested_fps == 5 else 0.05) # Validation KPI is 5% for all non 5 FPS rate
```

Comment:
This is fine but maybe we need an array `fps_threshold` or even `fps_delta_hz`?
Up to you...

Reply:
I thought about it and decided to go on this path, if we will have more than 1 exception we will create an array, currently it's the only exception